pyatlan/model/assets/core/a_i_model.py

```diff
@@ -129,6 +214,33 @@ class Attributes(AI.Attributes):
             default=None, description=""
         )  # relationship
 
+        @classmethod
+        @init_guid
+        def creator(
+            cls,
+            *,
+            name: str,
+            ai_model_status: AIModelStatus,
+            owner_groups: Optional[set[str]] = set(),
+            owner_users: Optional[set[str]] = set(),
```

Comment:
I'd recommend using type hints from the `typing` module (it's compatible with `Python 3.8+`).
```suggestion
            owner_groups: Optional[Set[str]] = set(),
            owner_users: Optional[Set[str]] = set(),
```

Reply:
Done